test/fixtures/controller.line/borderWidth/scriptable.js

```diff
@@ -6,18 +6,16 @@ module.exports = {
 			datasets: [
 				{
 					// option in dataset
-					data: [0, 5, 10, null, -10, -5],
-					pointBorderColor: '#0000ff',
-					pointBorderWidth: function(ctx) {
-						var value = ctx.dataset.data[ctx.dataIndex] || 0;
-						return value > 4 ? 10
-							: value > -4 ? 5
-							: 2;
+					data: [4, 5, 10, null, -10, -5],
+					borderColor: '#0000ff',
+					borderWidth: function(ctx) {
+						var index = (ctx.dataIndex === undefined ? ctx.datasetIndex : ctx.dataIndex);
+						return index === 0 ? 20 : 10;
```

Comment:
We may also need an explicit `pointBorderColor` for the first dataset else we can't see the different point border widths in the PNG (it's currently blue on blue):

![image](https://user-images.githubusercontent.com/3874900/54587088-59a5f200-4a1f-11e9-96bf-01242406e962.png)

I would replace `return index === 0 ? 20 : 10` by `return index % 2 ? 10 : 20;`